### PR TITLE
Add connect-src for snowplow, fix main docker

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: services/config/default.json [hunk 1]
```diff
@@ -61,12 +63,14 @@
...
     "PORT": "6379",
-    "PASSWORD": ""
+    "PASSWORD": "redis"
   },
   "EMAIL": {
-    "DEFAULT_ADMINS": [""]
+    "DEFAULT_ADMINS": [
+      ""
+    ]
   },
```
Comment (mhuseinov) [inline services/config/default.json:74]: These changes could break someone's local setup. Not an issue to fix but it could be avoided

Comment (dhaselhan) [reply services/config/default.json:74]: I'll make a post in #dev before I merge

File: portal-frontend/Dockerfile [hunk 2]
```diff
@@ -13,6 +13,8 @@ RUN npm ci
 # Copy the source code to the /app directory
 COPY . .
 
+ENV NODE_OPTIONS="--max-old-space-size=2048"
```
Comment (mhuseinov) [inline portal-frontend/Dockerfile:16]: did you run out of heap size?

**Review by mhuseinov — APPROVED** [review]
👍

File: portal-frontend/Dockerfile (hunk 2 above)
Comment (dhaselhan) [reply portal-frontend/Dockerfile:16]: yeah, had to give docker 10gb or Ram even with this set

**Pull request merged**